src/common/utils.js

```diff
@@ -123,13 +123,23 @@ const iconWithLabel = (icon, label, testid, iconSize) => {
 /**
  * Retrieves num with suffix k(thousands) precise to 1 decimal if greater than 999.
  *
- * @param {number} num The number to format.
- * @returns {string|number} The formatted number.
+ *  num The number to format.
```

Comment:
Why are the typings removed 🤔 ?

Reply:
I guess, I removed those typings by mistake, will add those again 👍